<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>740</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>740</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Algorithmic Trainding. Part 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="s21::GraphWidget" name="QGraphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>861</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>70</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Load data from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>270</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrintGraph">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1111</x>
      <y>270</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Print graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="secondWindowButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>30</y>
      <width>350</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Part2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1111</x>
      <y>170</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of points </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pointsNumberSpinBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>172</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
   </widget>
   <widget class="QLabel" name="file_name">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>650</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>file name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_to_calc">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>320</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="time_to_calc">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>370</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="price_result">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>470</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>result</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_price">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>420</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>calc price</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>s21::GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">view/graph_widget/graph_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
